### ISSUE TITLE
Remove skiplinks component

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#2732]
Remove it entirely for 1.0 
We'll add it back later, refactored (see #2730)

**Comment by mirkosoft** [comment]
@tobi-or-not-tobi @Xymmer should it be removed from here as well? It causes warnings after removing the component itself.
![image](https://user-images.githubusercontent.com/788092/58487355-b9c7bb80-8167-11e9-9f31-56f4ee40ebd8.png)

**Comment by tobi-or-not-tobi** [comment]
@mirkosoft did we drop the localisation keys from `common.ts` as well?

**Comment by mirkosoft** [comment]
No @tobi-or-not-tobi they're still present. Do you think they should be also removed?

**Comment by giancorderoortiz** [comment]
Please add QA steps.

**Comment by Pucek9** [comment]
open/close modals works as before

**Comment by mirkosoft** [comment]
QA Steps:
1. Open any page
2. Check the source for `<cx-skip-link>` tag/component call - it it's NOT present - we are good to go:
![image](https://user-images.githubusercontent.com/788092/59111079-a0312b80-8940-11e9-8622-d34f485d2ad7.png)

**Comment by Matejk00** [comment]
Tested and `<cx-skip-link>` tag not exists anymore.
